src/yum-plugins/enabled_repos_upload.py

```diff
@@ -2,11 +2,13 @@
 from yum import YumBase
```

Comment:
This import, the 'Yum' class in this file, and the 'TestYum' class in test_enabled_repos_upload.py can be removed

Reply:
> This import, the 'Yum' class in this file, and the 'TestYum' class in test_enabled_repos_upload.py can be removed - dead code!

will  look into this tomorrow